cmd/jimmsrv/main.go

```diff
@@ -38,15 +39,13 @@ func main() {
 //
 //nolint:gocognit // Start function to be ignored.
 func start(ctx context.Context, s *service.Service) error {
+	logLevel := os.Getenv("JIMM_LOG_LEVEL")
+	logLocal, _ := strconv.ParseBool(os.Getenv("JIMM_LOG_LOCAL"))
```

Comment:
And the same change for the env var name, because it's a bit unclear what does it mean. Maybe something like `JIMM_LOG_DEVELOPMENT`?